### PR TITLE
Add Travis CI

### PR DESCRIPTION
**Pull request by ribhavsharma**
Added travis.yml
Added Travis badge in README.md
[Task](https://codein.withgoogle.com/dashboard/task-instances/4862001500127232/)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 language: node_js
```
Comment (aashutoshrathi) [inline .travis.yml]: Make it, like this.
```yml
language: python
python:
  - 3.6
services:
- postgresql
```

Server is made in python not in node

File: README.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
+
+# lead-management-frontend
```
Comment (aashutoshrathi) [inline README.md]: This is unnecessary change, please make sure you revert it.

**Comment by aashutoshrathi** [comment]
Make travis.yml like this:
```yml
sudo: required
dist: trusty
language: node_js

node_js:
  - 8

cache:
  directories:
    - node_modules

```

**Comment by divyanshu-rawat** [comment]
what @aashutoshrathi  has mentioned, @ribhavsharma that should work well 👍

**Comment by aashutoshrathi** [comment]
@ribhavsharma updated comment please commit once again

**Comment by nikitadhiman** [comment]
@ribhavsharma conflicts present, please resolve at the earliest

**Comment by ribhavsharma** [comment]
conflicts resolved.

**Comment by nikitadhiman** [comment]
Closing this for now
